kujira/assetlist.json

```diff
@@ -361,6 +360,31 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/nstk.svg"
         }
       ]
+    },
+    {
+      "description": "Local Money",
+      "denom_units": [
+        {
+          "denom": "factory/kujira1swkuyt08z74n5jl7zr6hx0ru5sa2yev5v896p6/local",
+          "exponent": 0
+        },
+        {
+          "denom": "local",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/kujira1swkuyt08z74n5jl7zr6hx0ru5sa2yev5v896p6/local",
+      "name": "LOCAL",
```

Comment:
I think the name should be "Local Money" and that the description should be something longer